server/repository/src/db_diesel/activity_log_row.rs

```diff
@@ -50,6 +50,8 @@ pub enum ActivityLogType {
     StockOnHold,
     StockOffHold,
     Repack,
+    PrescriptionCreated,
+    PrescriptionDeleted,
```

Comment:
perhaps also `PrescriptionStatusVerified` and `PrescriptionStatusPicked` ?

<img width="330" alt="image" src="https://github.com/openmsupply/open-msupply/assets/9192912/ac474a7a-9a20-4fa2-94ee-6bd7f77b3dd2">